### PR TITLE
Update to ITU-R P.2108-1

### PR DESCRIPTION
**Pull request by aromanielloNTIA**
- Updates this software implementation from P2108-0 to P2108-1:
- Adds CSV test data for `AeronauticalStatisticalModel`, `HeightGainTerminalCorrectionModel`, and `TerrestrialStatisticalModel`
- Cleans up and improves Markdown formatting in README and LICENSE files
- Updates README for accuracy following software updates
- Changes .NET package namespace from `ITS.Propagation` to `ITS.ITU.PSeries`
- Updates .NET framework target from 4.72 to 4.8.1
- Adds unit tests for .NET wrapper
- Corrects previously erroneous definition of constant `DEFAULT_CLUTTER_HEIGHT__URBAN`
- Updates software package version to v1.0

### CODE REVIEW

**Comment by aromanielloNTIA** [comment]
Billy made the functional changes here- I updated some documentation and metadata things. I cannot "add my review" on GitHub since I opened the PR, but I have reviewed and tested the changes and I approve them. There are some minor things, such as the location of the testing data, which I'd change, but I will leave these to be handled by #6 following this PR and preceding a v1.0 release.

File: README.md [hunk 1]
```diff
@@ -28,109 +37,129 @@ Where site-specific values for the representative clutter height are not availab
...
 
- * [Recommendation ITU-R P.2108](https://www.itu.int/rec/R-REC-P.2108-0-201706-I/en)
- * [Report ITU-R P.2402](https://www.itu.int/pub/R-REP-P.2402)
+* [Recommendation ITU-R P.2108](https://www.itu.int/rec/R-REC-P.2108/en)
+* [Report ITU-R P.2402](https://www.itu.int/pub/R-REP-P.2402)
 
 ## Contact ##
 
-For questions, contact Billy Kozma, (303) 497-6082, wkozma@ntia.gov
+For questions, contact Billy Kozma, (303) 497-6082, <wkozma@ntia.gov>
```
Comment (wkozmaNTIA) [inline README.md:165]: I've been stripping out phone numbers from repos.  Both cause this number doesnt exist anymore (landline) and cause of spam.  So just use name/email for contact info

**Review by wkozmaNTIA — APPROVED** [review]
Docs look good.  As stated, @aromanielloNTIA  already check my software updates.

File: README.md (hunk 1 above)
Comment (aromanielloNTIA) [reply README.md]: Removed in afb17885ba493bc5cd1e4af837cdf57bf7449114

**Pull request merged**